### DISCUSSION TITLE
Error: crash loop back error after deploying the pod (sunbird-lms-service)

### DISCUSSION THREAD

**Discussion by imss2022** [Sunbird-Ed/Community#132] [Sunbird ED configuration discussion]
We are trying to move Sunbird User Management Service to Open Shift Platform. We are able to successfully build and deploy this service,  https://github.com/Sunbird-Lern/sunbird-lms-service/tree/release-5.0.0.  After deploying, the pod is getting into crash loop back error. While analyzing the error it is mentioned that the issue with docker file in the git repository. Kindly help out to solve the issue. corresponding screenshot is attached for further reference.
GitHub: https://github.com/Sunbird-Lern/sunbird-lms-service/tree/release-5.0.0

![error](https://user-images.githubusercontent.com/99241153/188579887-26421201-87fb-4b91-8254-995bfedc4d2d.png)

**Comment by beepdot** [comment]
Please try building the service once again and then redeploy. Many times this would solve the errors. Even after this it doesn't work, please provide the application logs in a txt file. You can attach the txt file in this thread.

**Reply by imss2022** [reply]
I tried redeploying also many times still facing the same issue.  i am attaching application logs below.
[sunbird-lms-service-55bf9c9978-swr8s-sunbird-lms-service.log](https://github.com/Sunbird-Ed/Community/files/9503125/sunbird-lms-service-55bf9c9978-swr8s-sunbird-lms-service.log)

**Reply by beepdot** [reply]
Please share the jenkins console log also from Build/Core/Learner job.

**Comment by imss2022** [comment]
i am sharing build log, we performed this build using openshift tekton pipeline.
[build.log](https://github.com/Sunbird-Ed/Community/files/9504048/build.log)

**Reply by chitranshu-keshav** [reply]
@amit-tarento @reshmi-nair Can you please get back on this

**Comment by beepdot** [comment]
@amit-tarento @reshmi-nair Can you please suggest what might be going wrong here?

**Comment by reshmi-nair** [comment]
@imss2022 This seems to be a deployment issue where the play dependencies are not being detected. Please make sure /lib from the unzipped distribution is in the classpath. If this is still not resolved then we can setup a call to discuss on the same.

**Reply by imss2022** [reply]
Still we are not able to resolve the issue, kindly  setup a call to discuss on the issue

**Comment by sharathkashyap** [comment]
@imss2022 Could you please help us to know if the issue was resolved

**Reply by sharathkashyap** [reply]
@imss2022 Will be locking this thread due to inactivity. Suggest you to raise the issue in separate thread if the issue is still unresolved.
